google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -1760,6 +1760,8 @@ public Rule apply(LifecycleRule lifecycleRule) {
     }
     if (logging != null) {
       bucketPb.setLogging(logging.toPb());
+    } else {
+      bucketPb.setLogging(Data.nullOf(Bucket.Logging.class));
```

Comment:
This can have unexpected behavior. Instead could you update setLogging() builder method to include this if else condition when `null` is provided?

Reply:
done